packages/env/README.md

```diff
@@ -27,3 +27,29 @@ class MyFactory {}
 
 const myEnv: DerivedEnvironment<MyFactory> = create(MyFactory)
 ```
+
+## Additional features
+
+### Dataset
+
+#### `rdf-ext` functionality
+
+The provided `DatasetCore` implementation extends it with methods matching the `rdf-ext` interface.
```

Comment:
not completely clear what "it" refers to. Maybe rephrase with something like "The provided `DatasetCore` implementation has additional methods..." or "The implementation of `DatasetCore` extends the base interface with..."